### DISCUSSION TITLE
私信如何用bot发送文件?

### DISCUSSION THREAD

**Discussion by vaecou** [Mrs4s/go-cqhttp#1470] [Q&A] [answered]
> 我试过发送zip文件给bot，然后让bot把离线文件给下载到缓存文件

> 缓存文件中有一个说明说可以使用CQ code码中的图片和语音发送，我试了好几个小时，但是我无法做到

> 在此向各位大佬提问，如何让bot在私信中发送文件

**Comment by RTAkland** [comment, marked as answer]
> gocq应该暂时无法实现

**Reply by vaecou** [reply]
应该是，看了所有的文档，确实没有实现的api

**Reply by vaecou** [reply]
但是群内的文件上传又有😥
